src/Analyzers/CSharp/Tests/MakeFieldReadonly/MakeFieldReadonlyTests.cs

```diff
@@ -2035,7 +2039,7 @@ public async Task ShouldNotWarnForDataMemberFieldsInDataContractClasses()
                 public class MyClass
                 {
                     [System.Runtime.Serialization.DataMember]
-                    private bool [|isReadOnly|];
+                    private bool [|isreadonly|];
```

Comment:
😕 Did you Find/Replace without case sensitivity here? Let's fix these.

Reply:
noice.